lib/resources/r4/provenance.yaml

```diff
@@ -0,0 +1,193 @@
+---
+name: Provenance
+field_name_base_url: http://hl7.org/fhir/R4/provenance-definitions.html#Provenance
```

Comment:
Same here, Aren't these supposed to be 'https' instead of 'http'?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/371/commits/a4119e15061c04918d4eb1a51f4e681b549723c5#diff-beb306fda480ca66704cb041f03846fbR3